test/optim/test_optim.py

```diff
@@ -1311,12 +1092,12 @@ def test_asgd(self):
             )
             self._test_complex_optimizer(
                 lambda params: ASGD(
-                    [params], maximize=True, weight_decay=0.9, foreach=foreach
+                    [params], maximize=True, weight_decay=0.1, foreach=foreach
```

Comment:
A quick GitHub grep shows that weight_decay of 0.1 is much much more common than 0.9, since people don't want to overregularize the weights.